lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -205,6 +218,7 @@ export async function initRepo({
       url,
       cloneSubmodules,
       fullClone: true,
+      authorization: opts.token,
```

Comment:
no, will already be handled via hostrule and is already added to opts.

you need to update the `getRepoGitUrl` function above to handle token auth.